src/components/Wizzard/ConvictedDetails/ConvictedDetails.js

```diff
@@ -0,0 +1,143 @@
+import classes from "./ConvictedDetails.module.scss";
+import Input from "../Input/Input";
+import {
+  WIZARD_PAGE_3_CONVICTED_DETAILS_WHAT_ERROR_LABEL,
+  WIZARD_PAGE_3_CONVICTED_DETAILS_WHAT_LABEL,
+  WIZARD_PAGE_3_CONVICTED_DETAILS_WHAT_PLACEHOLDER,
+  WIZARD_PAGE_3_CONVICTED_DETAILS_WHEN_LABEL,
+} from "../../../static";
+import PropTypes from "prop-types";
+
+/**
+ * Component for handling conviction details
+ * @param {object} props Input pairs validation and render handling logic
+ * @returns {JSX} Pairs of inputs for entering conviction details
+ */
+
+const ConvictedDetails = (props) => {
+  const {
```

Comment:
![image](https://user-images.githubusercontent.com/82144052/191108898-697d8d2c-b811-4ca9-a392-021f2f804cc6.png)

You can disable dates that are in the future to be selected.